service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -213,7 +213,7 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 		// Wait for chart-operator to be deployed. If it takes longer than
 		// the timeout the chartconfig CRs will be created during the next
 		// reconciliation loop.
-		b := backoff.NewConstant(30*time.Second, 5*time.Second)
+		b := backoff.NewConstant(20*time.Second, 10*time.Second)
```

Comment:
I would question if a resource should block for so long at all.

Reply:
@kopiczko But this is just once for all when TC is new. so I don't think this operator blocks so long. Moreover, you can't install any charts if we don't have chart-operator.

Reply:
Does it take 20s to install `chart-operator`? If not then we need to wait for the next loop anyway. So what's the point?

Reply:
Yes it's for new tenant clusters. chart-operator creates the chart CRD which is needed by the chart resource in app-operator.

We wait in this case so the chart CRs get created in the cluster in the same loop. Waiting for the next loop means the customer has to wait for the cluster to be fully usable.

This is only done to install chart-operator. For updates there is a chart CR and chart-operator updates itself.

Reply:
Maybe we should include this CRD/CR in the ignition?

Reply:
We think about that too, but chart CRD is mainly working with chart-operator, so we chose to install chart-operator for including this CRD/CR.

Reply:
And one more thing that I found; we can't install two CRDs in one controller :(

Reply:
We would need a way of rendering the CRDs from `apiextensions` in YAML which we don't have. Also now chart-operator installs the CRD like all the other operators which I find more intuitive. 

We need to make sure the operator doesn't retry for too long or too aggressively. If the install isn't going to succeed due to an error we should stop and retry in the next loop.

Reply:
@kopiczko Do you mind if we merge this change? Surely we can discuss further more in sig-operator.

Reply:
I don't mind. Sorry I should be more explicit. I have some concerns but it was like and I don't have enough insights to judge it. 👍